sdk/azidentity/azidentity.go

```diff
@@ -28,6 +28,15 @@ const (
 	defaultSuffix = "/.default"
 )
 
+const (
+	headerXmsDate       = "x-ms-date"
+	headerUserAgent     = "User-Agent"
+	headerURLEncoded    = "application/x-www-form-urlencoded"
+	headerAuthorization = "Authorization"
+	headerMetadata      = "Metadata"
+	headerContentType   = "Content-Type"
+)
+
```

Comment:
is there a benefit to put these in internal? Will other packages also use these?

Reply:
I thought the same as you, but look at the thread in this PR: https://github.com/Azure/azure-sdk-for-go/pull/15038. There's a point that it might take too long to make changes to `azcore` if we need to add new headers and then release a new version of the `internal` module. 

Still I think there's an argument to be made for having some of the more common ones in the `internal` module if it makes it easier to handle for other SDK devs.